pkg/renderer/helm_values.go

```diff
@@ -403,9 +403,6 @@ var (
 	publicValuesTemplate = template.Must(
 		helmTemplate.InitTemplate("values-public.yaml").Parse(publicValuesTemplateStr))
 
-	privateValuesTemplate = template.Must(
```

Comment:
should probably remove `privateValuesYamlPostgresTemplateStr` while you are at it.